packages/optimistic-oracle/src/proposer.js

```diff
@@ -493,7 +493,7 @@ class OptimisticOracleProposer {
       console.log("error", error);
       const message =
         error.type === "call" ? "Cannot settle for unknown reason☹️" : "Failed to settle proposal or dispute🚨";
-      this.logger.error({ at: "OptimisticOracleProposer#settleRequests", message, priceRequest, error });
+      this.logger.warn({ at: "OptimisticOracleProposer#settleRequests", message, priceRequest, error });
```

Comment:
Do you think it would make sense to check if it has already been settled to discard other issues? `oo.getState(...request) == Settled? "warn": "error"`

Reply:
we could do that...but I'm not totally sure that that will fix all situations where we get `error` on this. I think sometimes we send transactions on polygon that get stuck for multiple minutes in the mem pool due to polygon congestion/gas issues or whatever. in this case, I think we'd incorrectly log then.

Reply:
Ok, I see. I agree then that this solution is the simplest!